{
 "bad": [
  "exception_class",
  "exceptions.HttpClientError",
  "exceptions.HttpNotFoundError",
  "exceptions.HttpServerError",
  "exceptions.SerializerNotAvailable",
  "slumber.__init__.Resource",
  "slumber.__init__.ResourceAttributesMixin",
  "urlparse.urlparse",
  "urlparse.urlsplit",
  "urlparse.urlunsplit",
  "x",
  "x.content_types"
 ],
 "deps": {
  "exception_class": [],
  "exceptions": [],
  "json": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004"
  ],
  "posixpath": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004"
  ],
  "requests": [
   "google-cloud-bigquery-storage-core/conda-forge/noarch/google-cloud-bigquery-storage-core-2.0.0-pyh9f0ad1d_1",
   "requests/conda-forge/linux-64/requests-2.11.0-py35_0",
   "requests/conda-forge/noarch/requests-2.23.0-pyh8c360ce_2"
  ],
  "slumber": [],
  "urllib": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython",
   "python/conda-forge/linux-64/python-3.6.7-h381d211_1004"
  ],
  "urlparse": [],
  "x": [],
  "yaml": [
   "google-cloud-bigquery-storage-core/conda-forge/noarch/google-cloud-bigquery-storage-core-2.0.0-pyh9f0ad1d_1",
   "pyyaml/conda-forge/linux-64/pyyaml-5.3.1-py37h6b43d8f_2",
   "pyyaml/conda-forge/linux-64/pyyaml-5.4.1-py36h70b1f00_0"
  ]
 }
}